test/integration/targets/git/vars/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 
+source_dir:  "{{ output_dir }}/source"
```

Comment:
`msg` is only used once, so you can hardcode it there.

I would also not define variables for source_dir and destination_dir, but put the values directly.
Following the convention in `setup-local-repos.yml` I'd recommend to use `{{repo_dir}}/origin` for the "source" and `{{checkout_dir}}` for the "destination".